internal/operator/controller/contour/service.go

```diff
@@ -43,6 +43,62 @@ const (
 	// TODO [danehans]: Make proxy protocol configurable or automatically enabled. See
 	// https://github.com/projectcontour/contour-operator/issues/49 for details.
 	awsLbBackendProtoAnnotation = "service.beta.kubernetes.io/aws-load-balancer-backend-protocol"
+	// envoyServiceHTTPPort is the HTTP port number of the Envoy service.
+	envoyServiceHTTPPort = 80
+	// envoyServiceHTTPSPort is the HTTPS port number of the Envoy service.
+	envoyServiceHTTPSPort = 443
+	// envoyNodePortHTTPPort is the NodePort port number for Envoy's HTTP service. For NodePort
+	// details see: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport
+	envoyNodePortHTTPPort = 30080
+	// envoyNodePortHTTPSPort is the NodePort port number for Envoy's HTTPS service. For NodePort
+	// details see: https://kubernetes.io/docs/concepts/services-networking/service/#nodeport
+	envoyNodePortHTTPSPort = 30443
+	// awsProviderType is the name of the Amazon Web Services provider.
+	awsProviderType = "AWS"
```

Comment:
Not a problem for now, but we should look at defining these out into different files with an interface or something maybe. That would make adding new one's easy. 

But not needed for this specific PR.